src/parser/mod.rs

```diff
@@ -10150,7 +10163,35 @@ mod tests {
     fn test_replace_into_select() {
         let sql = r#"REPLACE INTO t1 (a, b, c) (SELECT * FROM t2)"#;
 
-        assert!(Parser::parse_sql(&GenericDialect {}, sql).is_err());
+        assert!(Parser::parse_sql(&GenericDialect {}, sql).is_ok());
+    }
+
+    #[test]
+    fn test_insert_into_select() {
+        let sql = r#"INSERT INTO t1 (a, b, c) (SELECT * FROM t2)"#;
+
+        assert!(Parser::parse_sql(&GenericDialect {}, sql).is_ok());
+    }
+
+    #[test]
+    fn test_insert_into_values() {
+        let sql = r#"INSERT INTO t1 (a) VALUES(1)"#;
+
+        assert!(Parser::parse_sql(&GenericDialect {}, sql).is_ok());
```

Comment:
Can you please put these tests in common with other similar tests, like this:

https://github.com/sqlparser-rs/sqlparser-rs/blob/2eabfb82083f97bb01883cd3c66c0b035343e3ba/tests/sqlparser_common.rs#L495-L501